assets/js/googlesitekit/notifications/components/common/Dismiss.js

```diff
@@ -33,12 +33,14 @@ export default function Dismiss( {
 	dismissLabel = __( 'OK, Got it!', 'google-site-kit' ),
 	dismissExpires = 0,
 	disabled,
+	onDismiss = () => {},
 } ) {
 	const trackEvents = useNotificationEvents( id );
 
 	const { dismissNotification } = useDispatch( CORE_NOTIFICATIONS );
 
-	const handleDismiss = () => {
+	const handleDismiss = async ( event ) => {
+		await onDismiss?.( event );
```

Comment:
Should this be a `?.` operator? There's a default defined, but I'm not actually clear on what happens if you await `null`.

I tested it and it _seems_ fine in node.js, so I guess this is okay 🤷🏻‍♂️ .